### ISSUE TITLE
Node.js Foundation Technical Steering Committee (TSC) Meeting 2018-05-23

### ISSUE THREAD

**Issue by mhdawson** [nodejs/TSC#544]
## Time

**UTC Wed 23-May-2018 22:00 (10:00 PM)**:

| Timezone      | Date/Time             |
|---------------|-----------------------|
| US / Pacific  | Wed 23-May-2018 15:00 (03:00 PM) |
| US / Mountain | Wed 23-May-2018 16:00 (04:00 PM) |
| US / Central  | Wed 23-May-2018 17:00 (05:00 PM) |
| US / Eastern  | Wed 23-May-2018 18:00 (06:00 PM) |
| London        | Wed 23-May-2018 23:00 (11:00 PM) |
| Amsterdam     | Thu 24-May-2018 00:00 (12:00 AM) |
| Moscow        | Thu 24-May-2018 01:00 (01:00 AM) |
| Chennai       | Thu 24-May-2018 03:30 (03:30 AM) |
| Hangzhou      | Thu 24-May-2018 06:00 (06:00 AM) |
| Tokyo         | Thu 24-May-2018 07:00 (07:00 AM) |
| Sydney        | Thu 24-May-2018 08:00 (08:00 AM) |

Or in your local time:
* http://www.timeanddate.com/worldclock/fixedtime.html?msg=Node.js+Foundation+Technical%20Steering%20Committee%20(TSC)+Meeting+2018-05-23&iso=20180523T22
* or http://www.wolframalpha.com/input/?i=10PM+UTC%2C+May+23%2C+2018+in+local+time

## Links

* Minutes Google Doc: <https://docs.google.com/document/d/1PorQJd2Ovi4e4ajXu_Ei_TCw7sHHucGxjNL5JEG80g0/edit>

## Agenda

Extracted from **tsc-agenda** labelled issues and pull requests from the **nodejs org** prior to the meeting.

### nodejs/node

* Nominating jdalton as collaborator [#20828](https://github.com/nodejs/node/issues/20828)
* doc: update the notable changes in 10.x [#20316](https://github.com/nodejs/node/pull/20316)
* doc: add documentation on porting from Buffer(arg) usage [#19827](https://github.com/nodejs/node/issues/19827)
* Tracking Issue for Runtime Deprecation of Buffer constructor [#19079](https://github.com/nodejs/node/issues/19079)

### nodejs/TSC

* Tracking issue for updating TSC on Board Meetings [#476](https://github.com/nodejs/TSC/issues/476)
* Strategic Initiatives - Tracking Issue [#423](https://github.com/nodejs/TSC/issues/423)

## Invited

* Anna Henningsen @addaleax (TSC)
* Сковорода Никита Андреевич @ChALkeR (TSC)
* Colin Ihrig @cjihrig (TSC)
* Daniel Bevenius @danbev (TSC)
* Franziska Hinkelmann @fhinkel (TSC)
* Gibson Fahnestock @gibfahn (TSC)
* Jeremiah Senkpiel @Fishrock123 (TSC)
* James M Snell @jasnell (TSC)
* Joyee Cheung @joyeecheung (TSC)
* Matteo Collina @mcollina (TSC)
* Michael Dawson @mhdawson (TSC)
* Myles Borins @MylesBorins (TSC)
* Ali Ijaz Sheikh @ofrobots (TSC)
* Rod Vagg @rvagg (TSC)
* Michaël Zasso @targos (TSC)
* Sakthipriyan Vairamani @thefourtheye (TSC)
* Timothy Gu @TimothyGu (TSC)
* Rich Trott @Trott (TSC)

## Notes

The agenda comes from issues labelled with `tsc-agenda` across **all of the repositories in the nodejs org**. Please label any additional issues that should be on the agenda before the meeting starts.

## Joining the meeting

Zoom link: https://zoom.us/j/611357642
Regular password

## Public participation

We stream our conference call straight to YouTube so anyone can listen to it live, it should start playing at **https://www.youtube.com/c/nodejs+foundation/live** when we turn it on. There's usually a short cat-herding time at the start of the meeting and then occasionally we have some quick private business to attend to before we can start recording & streaming. So be patient and it should show up.

Many of us will be on IRC in #node-dev on Freenode if you'd like to interact, we have a Q/A session scheduled at the end of the meeting if you'd like us to discuss anything in particular. @nodejs/collaborators in particular if there's anything you need from the TSC that's not worth putting on as a separate agenda item, this is a good place for that

**Comment by fhinkel** [comment]
I'm chairing this week.

**Comment by ofrobots** [comment]
I won't be able to make it this week – team event.

**Comment by Trott** [comment]
> I'd like to request that nodejs/i18n#77 be added to this meeting's agenda.

FWIW, I'm 👍 on the requested access here. If you're a TSC member that has an opinion on this, it would be great to use the :+1: or :-1: reactions so we can establish consensus or lack thereof ahead of the meeting.

**Comment by addaleax** [comment]
I’ll probably miss the meeting because it’s too late, but it’s probably worth mentioning https://github.com/nodejs/node/pull/20876 at some point. :)

**Comment by TimothyGu** [comment]
As usual, I can't make it to this time slot due to class.

**Comment by mhdawson** [comment]
I'm at a conference and will make it if I can, but quite possibly will not be able to.

**Comment by benjamingr** [comment]
@addaleax if you're up for it - I think it would be great to add the worker PR to the summit and do a session about it.

What I have in mind is getting a bunch of people to read the code together and review it so we can iron questions and problems out with you being accessible for questions.

**Comment by fhinkel** [comment]
@zeke why do those topics need to be added to the TSC agenda? Are they controversial? Have you tried resolving them on the issue tracker instead?

**Comment by Trott** [comment]
> @zeke why do those topics need to be added to the TSC agenda? Are they controversial? Have you tried resolving them on the issue tracker instead?

@fhinkel I agree that it *could* be resolved without needing to go to the meeting and I'm kind of hoping to do that in https://github.com/nodejs/TSC/issues/544#issuecomment-390859692 with the request for emoji reactions.

The approval needs to be done by an org owner, which is (basically) TSC. This is similar (although not identical, so kind of an undocumented grey area) to approving a bot. There was a request that was denied (by me) but I left a comment in the issue indicating that once all the information was gathered, the request could be escalated to TSC for approval.  See https://github.com/nodejs/i18n/issues/77#issuecomment-390853127. 

So, @zeke put it on the agenda because I told him to. :-D

IMO, it's appropriate to be on the agenda, even if all we do is ask that people who haven't weighed in to please do so. To be blunt, I think a pretty big percentage of the TSC will ignore or not see requests like this if they're not on the agenda. (We don't even bother listing `tsc-review` issues here anymore.)

**Comment by Trott** [comment]
Speaking of `tsc-review`: https://github.com/search?q=org%3Anodejs+label%3Atsc-review+is%3Aopen&type=Issues

Currently 9 open issues and pull requests labeled `tsc-review` including one dating as far back as November. 😱

**Comment by Trott** [comment]
(Removed `tsc-review` from https://github.com/nodejs/node/pull/17330 and left a comment. That was the one that was dating from November.)

**Comment by Trott** [comment]
Moderation Team update: No Moderation team actions taken this week!

By the way, those "no action taken" things are meant as "no actions taken in accordance with the Moderation Policy" so no one was blocked and that sort of thing.

However, a subset of the Moderation Team does talk (and there is a weekly meeting as well) and try to facilitate things and we notify each other to watch certain conversations and ask for feedback from each other on how we're handling things. So it's not like we're not doing anything, but it's mostly acting as individuals and not bringing to bear policy when it's not called for. (This paragraph is me speaking for myself only.)

@nodejs/moderation @nodejs/tsc @nodejs/community-committee

**Comment by mhdawson** [comment]
@Trott, not listing the TSC review items was probably lost when I went to automatically generating the issue.  If we think it is important to review in the meeting we can probably update to meeting generation tool to capture/add that list. (It is another reason to get to the point where we have template for both the meeting and minutes as we could then just adjust the templates to include a link and maybe the effort is better spent on that).  I know you'll probably not like this suggestion but we could immediately create a tracking issue so that we'd get a reminder to review every meeting.

**Comment by Trott** [comment]
>  I know you'll probably not like this suggestion but we could immediately create a tracking issue so that we'd get a reminder to review every meeting.

I think adding the review of `tsc-review` items to the agenda defeats the purpose, since the whole point of `tsc-review` was to label things that needed TSC attention but not meeting time.

I'm not that concerned about it, to be honest. It's easy enough to post a link in a comment like I did above. And although 9 tsc-review items is more than I'd like to see, it's also less than I expected to see, so there's good news too. :-D

**Comment by Trott** [comment]
(That said, adding that search link to the template would be :+1:.)

**Comment by danbev** [comment]
I'm going to have to pass tonights meeting, it's been a long day and a little too late for me.
